README.md

```diff
@@ -53,7 +53,27 @@ pip install merlin-models
 ```
 
 > Installing Merlin Models with `pip` does not install some additional GPU dependencies, such as the CUDA Toolkit.
-> When you run Merlin Models in one of our Docker containers, the dependencies are already installed.
+> It is best to use the highly optimized Docker containers with dependencies installed.
+> The following script can be used for installing dependencies locally, however due to the intricacies of individual setups, we are only able to help you troubleshoot any issues you might encounter when running Merlin Models inside one of the provided docker containers.
+
+```
+sudo apt update -y --fix-missing
+sudo apt install -y --no-install-recommends libexpat1-dev libsasl2-2 libssl-dev graphviz openssl protobuf-compiler software-properties-common
```

Comment:
For simplicity we might wish to keep this as minimal as possible. I think most of this script is not a dependency of merlin models or required for the installation or use of the library. 

The main one I can see is the `protobuf-compiler` which is required for the nvtabular installation.

And nvtabular is only installed currently if you request it with `pip install merlin-models[nvtabular]`.